### DISCUSSION TITLE
Is the send to menu still be used with Sandboxie Plus?

### DISCUSSION THREAD

**Discussion by bastik-1001** [sandboxie-plus/Sandboxie#3110] [Q&A Feedback] [answered]
Some time ago Sandboxie and Sandboxie Plus created entries in the Send To Menu. Now this does not seem to be the case, as newly created boxes don't show up there. So is that still an option to have (selected) boxes be present in that menu?

**Comment by bastik-1001** [comment]
Well something set entries to the Send To Menu, for some boxes, but those are not updated as newly created boxes don't get added to it.

**Reply by DavidXanatos** [reply, marked as answer]
Sandman UI does not use the send to menu at all, only SbieCtrl does.

**Reply by bastik-1001** [reply]
The reason I have those entries is, because I started SbieCtrl a long time ago (as it is bundled with the plus installer).

I assume that the more boxes a user has, the more of a clutter the Send To Menu becomes, if Sandman sets it by default. Do you find it reasonable to add a button or check-box to add boxes as needed by the user?
